assembl/static2/js/app/components/login/legalForm.jsx

```diff
@@ -0,0 +1,86 @@
+// @noflow
```

Comment:
why `@noflow` ? Can you add a comment in the file to explain why it's necessary please ?

Reply:
It's a typo due to IDE autocomplete, I ment @flow :) will fix